dockers/ubuntu-14.04/Dockerfile

```diff
@@ -73,7 +73,7 @@ ENV JAVA_HOME_8_X64=/usr/lib/jvm/zulu-8-amd64
 ENV JAVA_HOME=$JAVA_HOME_8_X64
 
 # Install SWIG
-RUN curl -sL https://sourceforge.net/projects/swig/files/swig/swig-4.0.2/swig-4.0.2.tar.gz/download -o swig.tar.gz \
+RUN curl -sLk https://sourceforge.net/projects/swig/files/swig/swig-4.0.2/swig-4.0.2.tar.gz/download -o swig.tar.gz \
```

Comment:
Same reason as it was in https://github.com/microsoft/LightGBM/pull/4648.
> Basically, I believe that the `nvidia/cuda` images (last updated 9/20/2021) are using a client somewhere that is using one or more certificates relying on the Let's Encrypt root certificate that expired 9/30/2021.

Reply:
makes sense! I think it's ok for applications like this.